### DISCUSSION TITLE
How to output both original and translated text?

### DISCUSSION THREAD

**Discussion by DrLucky** [openai/whisper#1876] [Q&A]
Whisper is quite amazing and I'm really enjoying its abilities. 
Currently I am using it to create subtitles of a foreign show and translate them into english.
Due to the length of time it takes to process, I would love to keep an output file of both the original text interpreted along with the english translations.
Is this possible currently?

**Comment by glangford** [comment]
> Due to the length of time it takes to process, I would love to keep an output file of both the original text interpreted along with the english translations.

You need to run whisper twice, once with `--task transcribe` then with `--task translate`.

**Reply by DrLucky** [reply]
wow, I hope that gets fixed at some point, that's a lot of extra unnecessary processing

**Comment by AfroDeivid** [comment]
Hello! Any update?

**Reply by glangford** [reply]
It's the same. But you can try the new turbo model to improve performance.

* https://github.com/openai/whisper/discussions/2363

> Whisper turbo was fine-tuned for two more epochs over the same amount of multilingual transcription data used for training large-v3, i.e. excluding translation data, on which we don’t expect turbo to perform well.
